easybuild/easyconfigs/m/MView/MView-1.67-foss-2022a-Perl-5.34.1.eb

```diff
@@ -0,0 +1,24 @@
+easyblock = "Tarball"
+
+name = 'MView'
+version = "1.67"
+
+homepage = 'http://sourceforge.net/projects/bio-mview/'
+description = """ MView reformats the results of a sequence database search or a 
+ multiple alignment, optionally adding HTML markup."""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+source_urls = [('http://sourceforge.net/projects/bio-mview/files/bio-mview/mview-%(version)s/', 'download')]
+sources = [SOURCELOWER_TAR_GZ]
+
+dependencies = [('Perl', '5.34.1')]
+
+sanity_check_paths = {
+    'files': ["bin/mview"],
```

Comment:
Is it possible to add `mview --help` (or similar) as a sanity check command?

Reply:
Done, thanks!